recipes-kernel/linux/linux-yocto-kitra710.bb

```diff
@@ -0,0 +1,20 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
+LINUX_VERSION = "4.1.15"
```

Comment:
no 4.4.19 kernel? The official Samsung kernel for artik710 has been upgraded from 4.1.15 to 4.4.19.